ConsoleApp/School/Teacher.cs

```diff
@@ -0,0 +1,10 @@
+namespace ConsoleApp.School;
+
+class Teacher : Person
+{
+    private int salary { get; set; }
```

Comment:
code convention violated here
```suggestion
    private int Salary { get; set; }
```